server/player_service.py

```diff
@@ -64,6 +64,15 @@ def clear_dirty(self):
 
     async def fetch_player_data(self, player):
         async with self._db.acquire() as conn:
+            result = await conn.execute(
```

Comment:
Could this be added to the query below (`sql = ...`)?

Reply:
Not very easily because both queries return a list. In order to get both lists in a single query I'm pretty sure you have to return the cartesian product of both lists which doesn't scale well at all.

Reply:
I see